README.md

```diff
@@ -83,7 +83,9 @@ tizen package -t wgt -o . -- .buildResult
    sdb connect YOUR_TV_IP
    ```
 4. If you are using a Samsung certificate, `Permit to install applications` on your TV using Device Manager from Tizen Studio. Or with sdb.
-   > TODO: Find a command
+   ```sh
+   sdb push ~/SamsungCertificate/<PROFILE_NAME>/*.xml /home/developer
+   ```
```

Comment:
There is `install-permit` command in the `tizen` CLI tool.
>   install-permit       Permit App installation for devices.

 Does it work? _I haven't tested._

Reply:
Thank you for taking the time to review this pull request. I apoligized for not getting back to you right away.

I did some more research and appearently the sdb command was a hacky workaround. At some point the install permit was broken in tizen.

I searched for documentation for install-permit but came up short.

I did however find some relevant info related to permissions [here](https://developer.tizen.org/development/tizen-studio/native-tools/managing-projects/device-manager#permitting). Specifically, this line:

- If your device has a distributor certificate that contains a DUID starting with 1.0#, you must give the device permission to install applications.

Otherwise, you don't need to give the device permission to install applications.

My DUID starts with 7 so I decided to run a fresh install flow with a new certificate, without permissions and the install completed without issue.

While I still had my TV connected I decided to run `tizen install-permit` to test what would happen. It gave me the following message:
`Please check the active certificate profile`

I'm guessing, based on the link above and the fact that my DUID starts with 7, that I can't test definitively if the sdb or install-permit command works properly.

Let me know how you want to proceed and I'll make the changes and update my PR. Many thanks.